fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: ''
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'trunk'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
```

Comment:
This was in order to get https://github.com/wordpress-mobile/release-toolkit/pull/361 in, in hope that it would reduce if not completely fix the `439 Too Many Requests` HTTP errors when downloading translations from GlotPress.

Alas, that didn't fix them, which is why there are only a handful of `strings.xml` files updated in that PR — as the others hit that 429 HTTP error. We're still working with GP.org folks to solve that ideally before next beta (and absolutely before next Wednesday's beta for localized testing and Friday's final build); but in the meantime that just means reduced translations in that intermediate beta, which is not that big a deal… for now.

Reply:
I fixed a small typo in this comment, which previously had `439` instead of `429`. 

I'm posting a note about it because I went on a 439 status code tangent... Found something cool: [HTTP has no official 439 status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses), however, the [folks at Etsy use 439 for "non-artisanal" requests](https://twitter.com/lozzd/status/380454656243412992?s=20&t=hEkONukHPICFDj24je3j_A) 🤔 😄